Lib/test/test_bytes.py

```diff
@@ -1599,6 +1599,8 @@ def test_extend(self):
         a = bytearray(b'')
         a.extend([Indexable(ord('a'))])
         self.assertEqual(a, b'a')
+        a = bytearray(b'abc')
+        self.assertRaises(TypeError, a.extend, 'def', msg="'str' object should be 'bytes'")
```

Comment:
`str` objects should never be `bytes` :)

I propose this wording instead: `Expected ...; got: str`

Reply:
Changed to 'Expected iterable of integers; got: str'.  I assume that this statisfies the request.